### DISCUSSION TITLE
Input text attribution towards BERT output vector

### DISCUSSION THREAD

**Discussion by mylar-pr** [jalammar/ecco#49] [Q&A]
I was wondering if it is possible to attribute input text with the BERT output vector of 768? 
I'm trying to find the attribution scores that show how much each word has affected the change in each of BERTs output vectors.

**Comment by jalammar** [comment]
Primary attribution is not yet supported for MLM models. I've created #64 to track it. If I understand correctly, you're looking for BERT attribution for token classification, correct?
